storage/merkle/tree.go

```diff
@@ -169,16 +214,20 @@ PutLoop:
 
 // Get will retrieve the value associated with the given key. It returns true
 // if the key was found and false otherwise.
-func (t *Tree) Get(key []byte) (interface{}, bool) {
-
-	// we start at the root again
-	cur := &t.root
-
-	// we use the given key as path again
-	path := bitset.Bytes(key[:])
+func (t *Tree) Get(key []byte) ([]byte, bool) {
+	if t.root == nil || t.keyLength != len(key) {
```

Comment:
I think `unsafeGet` can handle the empty tree case if I'm not wrong.
```suggestion
	if t.keyLength != len(key) {
```

Reply:
great observation 👍